packages/subgraph/tests/dao/utils.ts

```diff
@@ -334,15 +339,15 @@ export function getVotingToken(contractAddress: string, returns: string): void {
     .returns([ethereum.Value.fromAddress(Address.fromString(returns))]);
 }
 
-export function getIsUserWhitelisted(
+export function getIsUserAllowlisted(
   contractAddress: string,
   address: string,
   returns: boolean
 ): void {
   createMockedFunction(
     Address.fromString(contractAddress),
-    'isUserWhitelisted',
-    'isUserWhitelisted(address,uint256):(bool)'
+    'isUserAllowlisted',
```

Comment:
Something simpler like `isSigner()` related to the contract end.

Reply:
Changed this together with https://github.com/aragon/core/pull/62#discussion_r929646905. Closing this conversation.